### ISSUE TITLE
More detailed competitor page

### ISSUE THREAD

**Issue by AlexKatyukov** [thewca/wca-live#17]
Maybe good idea to add some information from WCA profile to WCA Live competitors page? For example, his personal bests. It helps speakers to prepare information about competitor before 3x3 finals or awards, for example.

**Comment by jonatanklosko** [comment]
There is a link to competitor WCA profile and it should be enough at this point. In the long term we could figure out a some UI displaying current personal bests.

**Comment by AlexKatyukov** [comment]
Yes, that is very nice, but it will be more convenient to have this at competitor WCA Live page without any additional tabs
